### ISSUE TITLE
RMSE loss units based on discrete azimuth bin numbers ?

### ISSUE THREAD

**Issue by pranaymanocha** [bingo-todd/WavLoc#1]
Hi! 
Thanks for the great repo. I was reading the code, and discovered that the RMSE azimuth loss is based on the **bin of the angle class**, and not on the angle prediction.  This is missing a factor of 5 to account for the discreteness in the azimuth space. Specifically, looking at lines: 250:254 in WaveLoc.py
```
    def _cal_azi_rmse(self, y_est, y):
        azi_est = tf.argmax(y_est, axis=1)
        azi = tf.argmax(y, axis=1)
        diff = tf.cast(azi_est - azi, tf.float32)
        return tf.sqrt(tf.reduce_mean(tf.pow(diff, 2)))
```
Does this make sense? I didnt try to run your code, or check very carefully - so there may be something I am missing here.

Thanks!!

**Comment by bingo-todd** [comment]
You are right, there should be a factor. But it is just a matter of scale. The output of this function is only used in logging not in optimizing the network.

**Comment by pranaymanocha** [comment]
Thanks for the reply! I was just trying to replicate the results from the paper, was getting values that were missing a scale factor from your repo, hence the comment.
